.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,23 @@
-# Change Type
+## Change Type (Feature, Chore, Bug Fix, RFC)
 
-* [ ] Feature
-* [ ] Chore
-* [ ] Bug Fix
+## Description
 
-# Change Level
+<!--
```

Comment:
Rather than ask all these questions in headings, asking them in an HTML comment gives folks the option to answer, or not, without something yelling at them about it.

Reply:
We can keep old structure without checkboxes (I also added Documentation)

[ ] Feature
[ ] Chore	
[ ] Bug Fix
[ ] Documentation